src/components/FindWallet/WalletPersonasSidebar.tsx

```diff
@@ -1,7 +1,13 @@
 // Libraries
 import React from "react"
-import { useTheme } from "@emotion/react"
-import styled from "@emotion/styled"
+import {
+  Box,
+  Flex,
+  Heading,
+  ListItem,
+  UnorderedList,
+  useColorMode,
+} from "@chakra-ui/react"
 
 // Components
 import Icon from "../Icon"
```

Comment:
This component is deprecated. Lets use `Icon` from Chakra.

[Here](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#update-dependencies) you can see more details on this update.